Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: test
+
+test:
+	docker-compose down
```

Comment:
Is there any reason to run `down` here? `run` will create a new container, and it's possible to `build` while containers are running

Reply:
I think I introduced this whilst I was still playing with the `postgres` image, and it's not necessary.